### PR TITLE
Drivetrain command

### PR DESCRIPTION
**Pull request by LiranBratt2121**
New support for a new `joystick` class
New support for a new `const` subclass
Created a new `command` that runs the dt

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -15,4 +15,8 @@ public static class DriveTrainConstants {
     public static final int kRightFrontPort = 3;
     public static final int kRightRearPort = 4;
   }
+
+  public static class Joystick{
+    public static final int kJoystickPort = 0; 
+  }
```
Comment (guy-av) [inline src/main/java/frc/robot/Constants.java:21]:
```suggestion

  public static class JoystickConstants {
    public static final int kJoystickPort = 0; 
  }
```

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -9,6 +9,11 @@
 import frc.robot.commands.ExampleCommand;
 import frc.robot.subsystems.DriveTrain;
 import frc.robot.subsystems.ExampleSubsystem;
+
+import javax.swing.border.EtchedBorder;
```
Comment (guy-av) [inline src/main/java/frc/robot/RobotContainer.java:13]: No

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -20,8 +25,8 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
-  private final DriveTrain m_driveTrain = new DriveTrain();
-
+  private static final DriveTrain m_driveTrain = new DriveTrain();
+  private static final XboxController joystick = new XboxController(RobotContainer.getDriveTrain().);
```
Comment (guy-av) [inline src/main/java/frc/robot/RobotContainer.java:29]: No

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -35,6 +40,13 @@ public RobotContainer() {
     configureBindings();
   }
 
+  public static DriveTrain getDriveTrain(){
+    return m_driveTrain; 
+  }
+
+  public static XboxController getJoystick(){
+    return joystick;
+  }
```
Comment (guy-av) [inline src/main/java/frc/robot/RobotContainer.java:49]: Remove these functions. No one should access them.

File: src/main/java/frc/robot/commands/DriveRobotArcade.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+
+public class DriveRobotArcade extends CommandBase {
+  /** Creates a new DriveRobotArcade. */
```
Comment (guy-av) [inline src/main/java/frc/robot/commands/DriveRobotArcade.java:11]: Redundant comment.
Other comments too

File: src/main/java/frc/robot/commands/DriveRobotArcade.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    double move = RobotContainer.getJoystick().getLeftY(); 
+    double rot = RobotContainer.getJoystick().getLeftX();
+    RobotContainer.getDriveTrain().arcadeDrive(-move, rot);
```
Comment (guy-av) [inline src/main/java/frc/robot/commands/DriveRobotArcade.java:25]: Don't use `getJoystick()` and `getDriveTrain()`

**Review by guy-av — CHANGES_REQUESTED** [review]
No

File: src/main/java/frc/robot/RobotContainer.java [hunk 7]
```diff
@@ -34,7 +41,6 @@ public RobotContainer() {
     // Configure the trigger bindings
     configureBindings();
   }
-
```
Comment (guy-av) [inline src/main/java/frc/robot/RobotContainer.java:37]: That's a good newline, separates two functions

File: src/main/java/frc/robot/RobotContainer.java [hunk 8]
```diff
@@ -22,19 +23,19 @@
 public class RobotContainer {
   private final DriveTrain m_driveTrain = new DriveTrain();
 
-  // The robot's subsystems and commands are defined here...
+  // The robot's subsystems and commands are defined here..
```
Comment (guy-av) [inline src/main/java/frc/robot/RobotContainer.java:26]: Unnecessary

**Review by guy-av — CHANGES_REQUESTED** [review]
Your commits are broken, one commit adds the lower half of `DriveArvade` and another commit adds the upper half of the same class. Since there aren't many changes in this PR, you can commit all changes together.

File: src/main/java/frc/robot/RobotContainer.java (hunk 8 above)
Comment (GlebiN) [reply src/main/java/frc/robot/RobotContainer.java]: removed some more comments in the last commit

**Comment by GlebiN** [comment]
We separated the commit of one file because one created it with adding things to constructor and the other added the functions inside the execute

**Review by guy-av — APPROVED** [review]

**Pull request merged**